src/client/Microsoft.Identity.Client/AppConfig/CacheOptions.cs

```diff
@@ -50,5 +61,12 @@ public CacheOptions(bool useSharedCache)
         /// </remarks>
         public bool UseSharedCache { get; set; }
 
+        /// <summary>
+        /// Token cache size limit in bytes.
+        /// </summary>
+        /// <remarks>
+        /// Once the limit is reached, the whole cache is cleared.
+        /// </remarks>
+        public long? SizeLimit { get; set; }
```

Comment:
Why is it nullable?

Reply:
In token cache we check, if it's null, then size limit doesn't apply.